index.js

```diff
@@ -21,6 +21,7 @@ class SignaturePad extends Component {
     onError: PropTypes.func,
     style: View.propTypes.style,
     penColor: PropTypes.string,
+    exportBackgroundColor: PropTypes.string,
```

Comment:
It accidentally snuck in there. Already removed it (https://github.com/DispatcherInc/react-native-signature-pad/pull/5/commits/e0553ea77811d7d1787c8e55ba6fd28e5ea03496)

Sorry about that.